src/console/main.ts

```diff
@@ -0,0 +1,27 @@
+import log from "electron-log";
+
+import { consoleDiscovery } from "./discovery";
+import { addMirrorConfig, startDiscovery, startMirroring, stopDiscovery } from "./ipc";
+import { mirrorManager } from "./mirrorManager";
+
+addMirrorConfig.main!.handle(async ({ config }) => {
+  mirrorManager.start(config);
+  return { success: true };
+});
+
+startMirroring.main!.handle(async ({ ip }) => {
+  mirrorManager
+    .startMirroring(ip)
+    .catch((err: Error) => log.info(`[Mirroring] Failed to start mirroring Wii @ ${ip}\n${err}`));
```

Comment:
I don't think we should be suppressing this error with `.catch`. It's better if we just throw the error so we can alert the frontend when there's an issue.

```suggestion
  await mirrorManager.startMirroring(ip);
```

Reply:
this was leftover from when I was trying to figure out why mirroring wasn't working (it was cause I forgot the port). Removed it.